src/pages/Home.js

```diff
@@ -27,6 +27,35 @@ function Home() {
           </button>
         </section>
       </div>
+      <section className="section__promo p-4">
+        <div className="promo__content">
+          <div className="content__head">
+            The Perfect Work from home accessories from <br /> Top Brands
+          </div>
+          <div className="content__subhead">
+            We are here to make good customers to conduct smooth and comfortable{" "}
+            <br />
+            work from home enivronment.
+          </div>
+        </div>
+        <div className="promo__images p-2">
+          <div className="promo__image">
+            <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTjNhE17d0VzkBRvRx4OIm0jjm_SR6ByRY4Zg&usqp=CAU" />
+          </div>
+          <div className="promo__image grid-span-2">
+            <img src="https://brandongaille.com/wp-content/uploads/2013/12/Bassett-Company-Logo.jpg" />
```

Comment:
I would suggest you use images from your own cdn, like cloudinary than from the source